project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject nomnom/caliban "1.0.2"
+(defproject nomnom/caliban "1.0.3-SNAPSHOT-1"
```

Comment:
Let's keep the snapshot version for now, we can move to stable once downstream libs like bunnicula.monitoring are fully up to date